projects/shikshalokam/src/lib/core-module/components/headers/headers.component.ts

```diff
@@ -28,4 +29,8 @@ export class HeadersComponent implements OnInit {
     this.logoutClick.emit('logout');
```

Comment:
Why are we doing like this??

Reply:
keycloak is in portal, that is why we are triggering on logout event in npm library and doing logout in portal